inc/cartridge.class.php

```diff
@@ -683,7 +683,7 @@ static function showForCartridgeItem(CartridgeItem $cartitem, $show_old = 0) {
             Html::showMassiveActions($massiveactionparams);
          }
          echo "<table class='tab_cadre_fixehov'>";
-         if (!$show_old) {
+         if ($show_old) {
```

Comment:
NOT ok with this one.

it invert the headers of the cartridges tables.
See: 
![image](https://user-images.githubusercontent.com/418844/42888711-19781dce-8aa9-11e8-9b4b-53a308780bcf.png)